lib/utils/emitDeprecationWarning.mjs

```diff
@@ -8,6 +8,7 @@ const CODES = {
 	GITHUB_FORMATTER: '004',
 	CONTEXT_FIX: '005',
 	RULE: '006', // either removal or renaming
+	REPORT_AMBIGUOUS_POSITION: '007',
```

Comment:
suggestion
nit

`AMBIGUOUS_POSITION` should be sufficient
I know you wanted to scope it to `report`.
You could leave it in if you think we may have other cases of _ambiguous positions_ elsewhere in the future.

Reply:
Let's keep this as-is.